install_config/aggregate_logging.adoc

```diff
@@ -197,7 +197,7 @@ ifdef::openshift-origin[]
 Otherwise, the template can be created with the following command:
 
 ----
-$ oc create -n openshift -f https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/files/examples/infrastructure-templates/origin/logging-deployer.yaml
+$ oc create -n openshift -f https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/deployment/README.md
```

Comment:
The previous link failed to stick around, and would have been the wrong format anyway (needs to be raw contents). Since this is origin-specific I think it's best to just get the latest from the origin repo.